style: add mono style for 1-bit displays

deadprogram opened the pull request:
This PR adds a new very basic style named `mono` for monochrome displays such as e-ink.

soypat commented on style/mono/mono.go:
```diff
@@ -0,0 +1,56 @@
+// NewTheme returns a new basic theme with the given properties.
+//
+// The default style is black text on a white background.
+func NewTheme[T pixel.Color](scale style.Scale, screen *tinygl.Screen[T]) *Mono[T] {
```
Maybe add a keyword to this function name? NewTheme sounds like a very generic constructor but this generates a roboto font theme specifically. Maybe `NewRobotoTheme`? `NewDefaultTheme` sounds ok too

deadprogram replied:
That's funny, because I was wondering if it should just be `New()` since you cannot create anything else in the `style` packages. :smile_cat:

aykevl replied:
`New` seems fine by me, I don't know why I originally used `NewTheme` instead.

soypat reviewed (commented):
Looks good but shooting a question your way about `NewTheme` naming

deadprogram commented:
I added another commit to rename the constructors to `New()`. I also renamed the "subtypes" to `Theme` to avoid stuttering and make it a little cleaner.

deadprogram commented:
@aykevl please merge! :smile:
